lib/pure/httpclient.nim

```diff
@@ -911,7 +911,9 @@ proc parseResponse(client: HttpClient | AsyncHttpClient,
         if line.len == le: httpError("Invalid headers - no colon after header name")
         inc(linei, le) # Skip the parsed header name
         inc(linei) # Skip :
-        if linei == line.len: httpError("Invalid headers - no header value after colon")
+        # Here linei is smaller or equal to line.len, so the slice below should work fine
+        # If we want to enforce the HTTP spec later, do this:
+        #if linei == line.len: httpError("Invalid headers - no header value after colon")
```

Comment:
It's not spec compliant, but I still commented this out for discussion - at least both Curl and Python seem to happily take headers that have a CRLF right after the header name + colon without any value and then show it as a header with an empty value.

Reply:
Makes me wonder if we should set up fuzzing tests for certain modules (to be run manually, at least).